tests/testthat/test-read_alevin.R

```diff
@@ -11,7 +11,8 @@ sample_ids = c("A1", "A2")
 
 test_that("reading salmon alevin data works", {
   sce <- read_alevin(alevin_dir,
-                     sample_id = sample_ids )
+                     sample_id = sample_ids,
+                     include_unspliced = FALSE)
```

Comment:
Can you add `expect_equal(sce@metadata$transcript_type, c("spliced"))` to this test?